nexus-sdk-js/src/views/ElasticSearchView/__tests__/ElasticSearchView.spec.ts

```diff
@@ -55,7 +56,7 @@ describe('ElasticSearchView class', () => {
   });
 
   // Skipping for now as it throw unhandled promise rejection errors.
-  describe.skip('query()', () => {
+  describe('query()', () => {
     const mockHttpPost = <jest.Mock<typeof httpPost>>httpPost;
```

Comment:
I'm not sure how this works exactly. We have `jest-fetch-mock` for mocking the fetch API. You can find an example here https://github.com/BlueBrain/nexus-webapp-commons/blob/master/nexus-sdk-js/src/Project/__tests__/Project.spec.ts#L157-L160

Reply:
I've updated to use jest-fetch-mock :)